MSAL/src/public/MSALInteractiveTokenParameters.h

```diff
@@ -70,6 +70,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithScopes:(NSArray<NSString *> *)scopes NS_DESIGNATED_INITIALIZER;
 
+#if TARGET_OS_IPHONE
+/*!
+ Modal presentation type for displaying authentication web content.
```

Comment:
Should this be above `presentationStyle`?

Reply:
Sure